lib/cloudregister/registerutils.py

```diff
@@ -1635,3 +1644,25 @@ def __replace_url_target(config_files, new_smt):
                     current_service_server,
                     new_smt.get_FQDN())
                 )
+
+
+# -----------------------------------------------------------------------------
+def __connection_check(smt_ip_addr, ip_format):
+    if not smt_ip_addr:
+        # in case RMT ever dropped IPv4 support
```

Comment:
What does not providing the `smt_ip_addr` argument have to do with whether or not the update servers have IPv4 addresses or not?